app/views/tracks/exercises/show/_iteration_section.html.haml

```diff
@@ -0,0 +1,120 @@
+          %p
+            = succeed(".") do
+              We’ve analysed your solution and have
+              %span.actionable= pluralize iteration.num_actionable_automated_comments, "recommendation"
+              - if iteration.has_non_actionable_automated_feedback?
+                and
+                %span.non-actionable= pluralize iteration.num_non_actionable_automated_comments, "additional comment"
+
+            You can either continue or address the recommendations first - your choice!
+        .status.passed Tests Passed
+      - else
+        .info
+          %h2 Your solution worked, but you can take it further…
+          %p
+            = succeed(".") do
+              We’ve analysed your solution and have
+              %span.actionable= pluralize iteration.num_actionable_automated_comments, "recommendation"
+              - if iteration.has_non_actionable_automated_feedback?
+                and
+                %span.non-actionable= pluralize iteration.num_non_actionable_automated_comments, "additional comment"
+
+            We suggest addressing the recommendations before proceeding.
+        .status.passed Tests Passed
+
+    - elsif iteration.status.to_sym == :non_actionable_automated_feedback
+      .info
+        %h2 Your solution looks great!
+        %p
+          We’ve analysed your solution and not found anything that needs changing.
+          We do have
+          %span.non-actionable= pluralize iteration.num_non_actionable_automated_comments, "additional comment"
+          that you might like to check.
+          - if exercise.practice_exercise?
+            Consider working with a mentor to make it even better.
+          %strong Great Job! 🎉
+      .status.passed Tests Passed
+    - else
+      .info
+        %h2 Your solution looks great!
+        %p
+          Your solution passed the tests and we've not got any recommendations.
+          - if exercise.practice_exercise?
+            You might want to work with a mentor to make it even better.
+          %strong Great Job! 🎉
+      .status.passed Tests Passed
+
+  = link_to Exercism::Routes.track_exercise_iteration_path(track, exercise, iteration), class: "iteration" do
+    = ReactComponents::Track::IterationSummary.new(iteration)
+    = graphical_icon 'chevron-right', css_class: 'action-icon'
+
+  = render ViewComponents::ProminentLink.new("See all of your iterations", track_concepts_path(track))
+
+  .next-steps
+    .community-solutions
+      = graphical_icon "graphic-community-solutions", css_class: "header-icon"
+      %h3 Learn from others' solutions
+      %p Explore their approaches to learn new tips and tricks. Discover popular solutions to this exercise.
+      = link_to "View community solutions", "#", class: 'btn'
+    .mentoring
+      = graphical_icon "graphic-mentoring-screen", css_class: "header-icon"
+      %h3 Get mentored by a human
+      %p On average, students iterate a further 3.5 times when mentored on a solution.
```

Comment:
Does this need a TODO?

Reply:
I'm going to go through all the copy everywhere in templates before we launch, so I tend not to add them to individual places.